docs/kyma/04-13-local-reinstallation-deprecated.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Reinstall Kyma (Scripts)
+type: Installation
+---
+
+>**WARNING:** This document will be removed with Kyma release 1.4. For local installation using Kyma CLI, see [this document](/root/kyma/#installation-reinstall-kyma).
```

Comment:
1. We don't have "WARNING" notes - "CAUTION" notes are available, though.
2. Please re-think the deprecation message. I don't think that stating the exact cutoff release is the right way to go.